state/runtime/evm/instructions.go

```diff
@@ -1191,10 +1191,16 @@ func (c *state) buildCallContract(op OpCode) (*runtime.Contract, uint64, uint64,
 	addr, _ := c.popAddr()
 
 	var value *big.Int
-	if op == CALL || op == CALLCODE {
+	if op == CALL || op == CALLCODE || op == AUTHCALL {
 		value = c.pop()
 	}
 
+	var extValue *big.Int
```

Comment:
Can you split everything for AUTH/AUTHCALL into a separated PR?

Reply:
Moved https://github.com/0xPolygon/polygon-edge/pull/1291